send_sms.py

```diff
@@ -0,0 +1,31 @@
+import requests
+import json
+import general
+
+URL = 'http://www.way2sms.com/api/v1/sendCampaign'
+
+
+# get request
+def sendPostRequest(reqUrl, apiKey, secretKey, useType, phoneNo, senderId, textMessage):
```

Comment:
Can you please change this to `send_post_request`? 
I've tried to follow the PEP8 naming conventions. See https://www.python.org/dev/peps/pep-0008/#function-and-variable-names